internal/ssrf/ssrf.go

```diff
@@ -1,20 +1,17 @@
 package ssrf
```

Comment:
It looks like that [vulnerability is disabled](https://user-images.githubusercontent.com/93549255/142511069-13e75175-cb78-4b5c-82ae-4428438b287a.png) in the new change. We have a SSRF handler and template though, I guess it just didn't make it into the `routes.json`.

Reply:
fixed